### ISSUE TITLE
Pasting content with <figure> in it crashes editor: Uncaught Invariant Violation: Unknown DraftEntity key: null

### ISSUE THREAD

**Issue by ForteDell** [jpuri/react-draft-wysiwyg#824]
E.g. when a user copies a page with a figure tag in the html to show an image,
User expects to see the page in the rich text editor.
But the editor crashes.

**Comment by ForteDell** [comment]
Hello: Fixed it:

in the <Editor
	handlePastedText={this.handlePastedText}
/>




	/**
	 * 
	 * @param text
	 * @param html
	 */
	handlePastedText = (text, html) => {
		//If copied text contains something we don't like, we can adjust it. 
		if (text.indexOf('BAD WORD') != -1) {
			// we'll add a message for the offending user to the editor state
			const newContent = Modifier.insertText(
					this.state.editorState.getCurrentContent(),
					this.state.editorState.getSelection(),
					'GOOD WORD'
			);

			// update our state with the new editor content
			this.onChange(EditorState.push(this.state.editorState, newContent, 'insert-characters'));
			return true;
		}
		//Editor can't deal with <figure> tags. We replace figure tags with divs. Then the content is pasted like a normal img.
		if (html) {
			html = html.replace(/(<\/?)figure((?:\s+.*?)?>)/g, '$1div$2');

			const newContent = Modifier.insertText(
					this.state.editorState.getCurrentContent(),
					this.state.editorState.getSelection(),
					html
			);
		}
		return false;
	}

**Comment by ForteDell** [comment]
Ah, somehow this then breaks a paste with a normal img tag in it. I'm lost.

**Comment by Rody-Kirwan** [comment]
Yes - I'm also getting this error when I copy and paste both img and text.
Steps
 - add text and image to editor
 - copy text and image
 - paste

Error:

invariant.js:42 Uncaught Invariant Violation: Unknown DraftEntity key: null.

**Comment by tcg** [comment]
I think I'm having this issue when loading any content (from HTML) that contains a `figure` tag. I can't yet set up a reduced environment/test, but it seems to me like `figure` consistently breaks the editor. 

<img width="590" alt="DevTools Console output showing Error: Unknown DraftEntity key." src="https://user-images.githubusercontent.com/575773/62794358-807dc700-ba99-11e9-9eb7-dcfb6532cbfd.png">

**Comment by tcg** [comment]
I've used Create React App to put together a very basic example. 

Here's the relevant bit: https://github.com/tcg/react-draft-wysiwyg-figure-bug-example/blob/master/src/App.js

There's a production build of this at https://blissful-bose-7e64f9.netlify.com/ but you can't see the dev results, etc. Best would be to clone + run that project so you can see the unminified + dev errors, like so: 

![image](https://user-images.githubusercontent.com/575773/62797884-78765500-baa2-11e9-98bd-7bb35168ad72.png)

**Comment by cuongdevjs** [comment]
Has there any solution to this problem?

**Comment by tanduong** [comment]
Switch to another editor.
